api-client/lib/api/extensions/get_into_teaching_api_client/api_client.rb

```diff
@@ -62,20 +74,35 @@ def object_to_hash(obj)
         end
       end
 
-    private
+      private
 
       def format_date_times(params = {})
         params.transform_values do |value|
           case value
-            when DateTime, Time
-              value.strftime(API_DATE_TIME_FORMAT) 
-            when Date
-              value.strftime(API_DATE_FORMAT)
-            else
-              value
+          when DateTime, Time
+            value.strftime(API_DATE_TIME_FORMAT)
+          when Date
+            value.strftime(API_DATE_FORMAT)
+          else
+            value
           end
         end
       end
+
+      def error_breaks_circuit?(exception, handler)
```

Comment:
This method name doesn't make sense anymore; when we wrote it originally I thought we were going to return a `bool` but it actually either re-raises or passes to the circuit breaker handler. I would just go with `handle_error`.